llvm/test/Transforms/LoopVectorize/ARM/arm-ieee-vectorize.ll

```diff
@@ -108,7 +108,7 @@ for.end.loopexit:                                 ; preds = %for.body
   br label %for.end
 
 for.end:                                          ; preds = %for.end.loopexit, %entry
-  %Red.0.lcssa = phi i32 [ undef, %entry ], [ %add.lcssa, %for.end.loopexit ]
+  %Red.0.lcssa = phi i32 [ 0, %entry ], [ %add.lcssa, %for.end.loopexit ]
```

Comment:
we could remove the phi node alltogether, by removing the unneeded check in the entry block